src/main/resources/se/diabol/jenkins/pipeline/DeliveryPipelineView/configure-entries.jelly

```diff
@@ -27,6 +27,10 @@
             <f:select/>
         </f:entry>
 
+        <f:entry title="Max number of pipelines" field="maxNumberOfJobs" default="-1">
+            <f:textbox/>
+        </f:entry>
```

Comment:
@patbos I don't think limiting the number of pipelines makes sense without the sorting. If you have a static order why not just remove the pipelines that are not shown from the pipeline view configuration? But I'm open to either solution.